zul/src/archive/web/js/zul/wgt/Notification.js

```diff
@@ -161,7 +163,7 @@ zul.wgt.Notification = zk.$extends(zul.wgt.Popup, {
 
 		var pzcls = this.$s('pointer'),
 			n = this.$n(),
-			refn = ref.$n(),
+			refn = ref.$n();
 			dir = this._dir,

```

Comment:
global?

Reply:
sorry, I test the code before and try to change back, but not change it completely.